src/services/sessions.js

```diff
@@ -1618,6 +1715,18 @@ module.exports = class SessionsHelper {
 
 			const processDbResponse = utils.processDbResponse(sessionDetails, validationData)
 
+			// Cache the session details for future requests (only for simple numeric id requests)
+			if (utils.isNumeric(id) && (!userId || !queryParams?.get_mentees)) {
+				// Cache the processed session without user-specific data
+				const cacheData = { ...processDbResponse }
+				delete cacheData.share_link
+				delete cacheData.mentee_password
+				delete cacheData.mentor_password
+
+				await cacheHelper.sessions.set(tenantCode, orgCode, sessionDetails.id, cacheData)
```

Comment:
@sumanvpacewisdom in some other placess ur seting db data directly to sessions,  
We need all the sessions info , use only details API to set the sessions cache, dont set in session create/update